stable/prometheus/values.yaml

```diff
@@ -763,7 +763,18 @@ pushgateway:
     servicePort: 9091
     type: ClusterIP
 
-## alertmanager ConfigMap entries
+sidecar:
+  image:
+    repository: kiwigrid/k8s-sidecar
+    tag: 0.0.5
```

Comment:
I would suggest upgrading the version to `0.0.6` since it adds the feature to use the current namespace (in which the sidecar runs) for the ConfigMap lookup. This is necessary to be able to run the sidecar (and the whole chart) in environments with limited privileges. If a specific namespace is desired for the search, it can be specified through the `Namespace` env variable. It a cluster-wide search is needed (all namespaces). The value `ALL` can be set.

Reply:
That is a good idea. I'm a bit busy recently but I'll change the version asap.